src/components/grid-aware/Footer/Footer.module.css

```diff
@@ -0,0 +1,257 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-rows: 120px 380px 78px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    text-align: center;
+  }
+
+  .gridParent {
+    column-gap: var(--grid-column-gap);
+    display: grid;
+    grid-template-rows: 190px auto auto;
+  }
+}
+
+/* Top Row */
+
+.gridTopRow {
+  align-content: center;
+  background-color: var(--color-black);
+  display: flex;
+  grid-column: 1 / span 12;
+  grid-row: 1;
+  justify-content: center;
+}
+
+.subscribeContainer {
+  align-items: baseline;
+  align-self: center;
+  display: flex;
+  justify-content: center;
+}
+
+.subscribeMessage {
+  color: var(--color-white);
+  font: var(--font-subtitle);
+  padding-right: 20px;
+  width: 340px;
+}
+
+.gridTopRow a {
+  padding: 14px 40px;
+}
+
+.gridTopRow a:hover {
+  top: 0;
+}
+
+.formInput {
+  display: flex;
+  flex-direction: row;
+}
+
+.formInput input {
+  border: 0;
+  color: var(--color-black);
+  font: var(--font-body-small);
+  padding: 16px 0 15px 17px;
+  width: 291px;
+}
+
+@media (--tablet-and-down) {
+  .gridTopRow {
+    padding-bottom: 43px;
+    padding-top: 44px;
+  }
+
+  .subscribeMessage {
+    padding-bottom: 22px;
+  }
+
+  .subscribeContainer {
+    display: unset;
+    flex-direction: column;
+  }
+
+  .subscribeMessage {
+    padding-right: 0;
+  }
+
+  .formInput {
+    display: unset;
+  }
+
+  .formInput input {
+    width: 155px;
```

Comment:
Perhaps instead of fixing this to a specific width, you want this to grow to fill the available space depending on the screen size? The text box needs to be narrower on smaller screens, but it would be beneficial to have more space on slightly larger mobile screens.

We probably still want some max width, especially because we don't have a real tablet design, so we don't want it to stretch to the full width of a tablet-sized screen.

Reply:
I may need to discuss this on call with you

Reply:
Changed component to be the width of parent